multi-agent/config/app_config.py

```
@@ -14,7 +16,8 @@ class AppConfig(SharedConfig):
     hostname: str = "0.0.0.0"
     port: str = "8002"
     version: str = "1.0.0"
-    admin_allowed_users: list = []  # Populate with user_ids (usernames) to grant admin access
+    admin_allowed_users: List[str] = []  # Set via env: ADMIN_ALLOWED_USERS="user1,user2"
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)